R/kde1d_methods.R

```diff
@@ -59,6 +70,7 @@ pkde1d <- function(q, obj) {
         q <- q[[1]]
     if (!is.ordered(q))
         stopifnot(!is.factor(q))
+    check_boundary_violations(q, obj$xmin, obj$xmax)
```

Comment:
Shouldn't it be 0 for elements of q <= obj$xmin and 1 for elements >= obj$xmax?

Reply:
agreed